#	Removing unused function broadcast_uevent() as it is trying to kree_skb when CONFIG_NET is not defined

diff --git a/lib/kobject_uevent.c b/lib/kobject_uevent.c
--- a/lib/kobject_uevent.c
+++ b/lib/kobject_uevent.c
@@ -408,15 +408,9 @@ int broadcast_uevent(struct sk_buff *skb
 	kfree_skb(skb);
 	return err;
 }
-#else
-int broadcast_uevent(struct sk_buff *skb, __u32 pid, __u32 group,
-		     gfp_t allocation)
-{
-	kfree_skb(skb);
-	return 0;
-}
+
+EXPORT_SYMBOL_GPL(broadcast_uevent);
 #endif
-EXPORT_SYMBOL_GPL(broadcast_uevent);
 
 #if defined(CONFIG_NET)
 static int uevent_net_init(struct net *net)
